content/en/docs/how-tos/adding-a-new-secret-to-ci.md

```diff
@@ -47,6 +48,14 @@ for details.
 The propagation of secret contents is scheduled immediately after they are
 added or modified and should be completed within 30m.
 
+To use the secret in a job step, export the secret. For example, in `your-build-step-command.sh`:
+
+```bash
+AUTH_TOKEN=$(cat /tmp/vault/my-secret/MY_SECRET_TOKEN)
+
+export AUTH_TOKEN
+```
+
```

Comment:
I would move this after "for details" in line 46 and modify it like:

Suppose the `mount_path` for the above secret `my-secret` is `/var/run/my-data` defined in a step. The script in the step can then access `my_secret_value` via the file `/var/run/my-data/my_secret_key`.


```suggestion

```